Doc/howto/unicode.rst

```diff
@@ -531,7 +533,8 @@ Marc-André Lemburg gave `a presentation titled "Python and Unicode" (PDF slides
 <https://downloads.egenix.com/python/Unicode-EPC2002-Talk.pdf>`_ at
 EuroPython 2002.  The slides are an excellent overview of the design of Python
 2's Unicode features (where the Unicode string type is called ``unicode`` and
-literals start with ``u``).
+literals start with ``u``). (Notice: Python 2 is no longer supported, just keep it
```

Comment:
Better: (Note: Python 2 is no longer supported. We just keep this reference for historic reasons.)